cookbooks/aws-parallelcluster-config/resources/manage_ebs.rb

```diff
@@ -90,8 +90,9 @@
       options "_netdev"
       pass 0
       action %i(mount enable)
-      retries 10
+      retries 1
```

Comment:
Here and elsewhere.
Why should we decrease the number of retries?

Reply:
I decrease the number for faster testing the change, the PR was not ready for review at that time and it was a draft change